bin/oref0-setup.sh

```diff
@@ -255,6 +255,32 @@ function add_to_crontab () {
     (crontab -l; crontab -l |grep -q "$1" || echo "$2" "$3") | crontab -
 }
 
+function request_stop_local_binary () {
+    if [[ -x /usr/local/bin/$1 ]]; then
+        if pgrep -x $1 > /dev/null; then
+            read -p "Need to stop $1 to complete installation. OK? [Y]N" -r
+            if [[ $REPLY =~ ^[^Nn]*$ ]]; then
```

Comment:
Is this logic correct?  I'm not sure what the second ^ does, but I think we want this to be something like `if [[ $REPLY =~ ^[Yy]*$ ]]; then`, right?

Reply:
The `^` tells it to check for "not 'N' or 'n'" - my thought was to have "yes" be the default so users could just hit enter. I'll explore the `prompt_yn` function, which looks like a better solution.